types/electra/attestation.yaml

```diff
@@ -0,0 +1,33 @@
+Electra:
+  IndexedAttestation:
+    type: object
+    description: "The [`IndexedAttestation`](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#indexedattestation) object from the CL spec."
+    properties:
+      attesting_indices:
+        type: array
+        maxItems: 131072
+        description: "Attesting validator indices"
+        items:
+          $ref: "../primitive.yaml#/Uint64"
+      signature:
+        $ref: '../primitive.yaml#/Signature'
+        description: "The BLS signature of the `IndexedAttestation`, created by the validator of the attestation."
+      data:
+        $ref: '../attestation_data.yaml#/AttestationData'
+
+  Attestation:
+    type: object
+    description: "The [`Attestation`](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#attestation) object from the CL spec."
```

Comment:
same everywhere really, lock to the alpha.2 for these instead of `/dev`

Reply:
done, used the alpha version for now